CHANGES

```diff
@@ -9,6 +9,11 @@
 
  Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
 
+  *) Hundreds of improvements were made to the documentation, including:
+     adding missing command flags, better style conformance, documentation
+     of internals, etc.
+     [Many]
```

Comment:
„Hundreds of improvements...“ by „[Many]“?  Sounds a little bit exaggerated and vague at the same time. 

I‘d suggest:

„A lot of improvements...“ by „[Rich Salz and others]“. (or „e.a.“)

Reply:
If you think that others made outstanding contributions to the documentation and should be mentioned too, feel free to add them.

Reply:
I made\ over 2000 changes via ```git diff master | grep '^+' | wc -l``` so I reworded it and put myself as the author.  Other folks should add themselves.

Reply:
Oh, I see. I wasn't aware that the "Hundreds of improvements" were measured in line counts ;-)

Reply:
> put myself as the author

You truly deserve the credits IMHO. Nobody else put so much effort into cleaning up the documentation as you did recently.